plugins/inputs/kafka_consumer/kafka_consumer_test.go

```diff
@@ -484,58 +484,24 @@ func TestKafkaRoundTripIntegration(t *testing.T) {
 	}{
 		{"connection strategy startup", "startup", []string{"Test"}, nil, config.Duration(0)},
 		{"connection strategy defer", "defer", []string{"Test"}, nil, config.Duration(0)},
-		{"topic regexp", "startup", nil, []string{"T*"}, config.Duration(5 * time.Second)},
```

Comment:
Any reason you removed this one?

Reply:
Yes on purpose. The previous test containers would set up a test topic, so we could try out the regex option instead of specifying specifically a new Test topic. Since no topics exist anymore, we cannot do this scenario.